docs/serverless/08-02-expose-function.md

```diff
@@ -75,13 +75,13 @@ Follows these steps:
     ```bash
     curl https://$NAME.$DOMAIN
     ```
-   
-   >**NOTE:** If you are using Minikube, make sure that you've added the endpoint to the Minikube IP entry in the `/etc/hosts` file. 
+
+   >**NOTE:** If you are using Minikube, make sure that you've added the endpoint to the Minikube IP entry in the `/etc/hosts` file.
```

Comment:
```suggestion
   >**NOTE:** If you are using Minikube, make sure that you've added the domain name to the Minikube IP entry in the `/etc/hosts` file.
```
It's not an endpoint, it's a hostname/a domain name. See also [this comment](https://github.com/kyma-project/kyma/pull/9296/files#r481078607).